src/linalg.jl

```diff
@@ -157,6 +157,20 @@ function _mul!(C::BlockDiagonal, A::BlockDiagonal, B::BlockDiagonal, α::Number,
     return C
 end
 
+function LinearAlgebra.lmul!(B::LowerTriangular{<:Any,<:BlockDiagonal}, vm::StridedVecOrMat)
```

Comment:
Could we add a comment here linking the issue? It's somewhat unusual to be implementing internal functions from LinearAlgebra, so we should provide a reason for the reader.

Reply:
Will do

Reply:
Resolved by https://github.com/invenia/BlockDiagonals.jl/pull/119/commits/d59b578df6e81b74f32c3a43c8bd5e06af4cc0df